fuzz/make_corpus.py

```diff
@@ -0,0 +1,23 @@
+import json
+import sys
+import os.path
+
+"""
+Creates a directory containing seed inputs from a json file.
+Usage:
+- pass the resulting corpus directory path as the first argument
+- pass the json file path to make the corpus from as the second argument
+"""
+corpus_dir = sys.argv[1]
```

Comment:
This files looks like you outsiders might want to execute it? Then it needs more robust parameter handling, like using `argparse`.